package.json

```diff
@@ -36,8 +36,8 @@
     "@fortawesome/free-regular-svg-icons": "^5.13.0",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/react-fontawesome": "^0.1.9",
-    "@prisma/cli": "^2.0.0-beta.8",
-    "@prisma/client": "^2.0.0-beta.8",
+    "@prisma/cli": "^2.1.0",
```

Comment:
why not 2.1.3?

Reply:
because I didn't know the patch version, but yarn resolved the version to `2.1.3` anyways